### ISSUE TITLE
Failure to create GSPS when reference DICOM uses "MultiValue" for WindowCenter, WindowWidth

### ISSUE THREAD

**Issue by delton137** [ImagingDataCommons/highdicom#210]
When I go to create a GSPS using hd.pr.GrayscaleSoftcopyPresentationState() I get an error for certain DICOM where WindowWidth and WindowCenter are not floats in the reference DICOM. 

Here's the error I get:
```
~/anaconda3/lib/python3.8/site-packages/highdicom/pr/content.py in _get_softcopy_voi_lut_transformations(referenced_images)
   1456 
   1457             if has_width:
-> 1458                 by_window[(
   1459                     ref_im.WindowWidth,
   1460                     ref_im.WindowCenter,

TypeError: unhashable type: 'MultiValue'
```

 I fixed it using the following code: 

``` Python
        if isinstance(image_dataset.WindowWidth, pydicom.multival.MultiValue):
            image_dataset.WindowWidth = image_dataset.WindowWidth[0]
        if isinstance(image_dataset.WindowCenter, pydicom.multival.MultiValue):
            image_dataset.WindowCenter = image_dataset.WindowCenter[0]
```

This fix worked for two offending DICOM, but there is one more where I'm still getting the error. To get rid of the error there, I created the GSPS with my own windowing settings (using the argument voi_lut_transformations=[SoftcopyVOILUTObject]).

**Comment by CPBridge** [comment]
Thanks @delton137 ! I had a quick look and I think this is a simple fix. Can you try the branch in #211 and see whether that fixes the issue?

For now, the best solution is probably to manually specify the `voi_lut_transformations`, as you suggest.

**Comment by hackermd** [comment]
Why would these attributes have a multiplicity of greater than one?

**Comment by pieper** [comment]
>Why would these attributes have a multiplicity greater than one?

It's not uncommon for an image to have multiple WW/WC values in the header.  They could be soft tissue and lung windows for example.

**Comment by CPBridge** [comment]
[C.11.2.1.2.2](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.11.2.html#sect_C.11.2.1.2.2), with reference to WindowCenter and WindowWidth:

> If multiple values are present, both Attributes shall have the same number of values and shall be considered as pairs. Multiple values indicate that multiple alternative views may be presented.

**Comment by delton137** [comment]
I just tested @CPBridge's fix (#211) and it works! Yes, it seems multiple window settings may be saved in the DICOM.

**Comment by delton137** [comment]
Update: Chris's fix #211 worked for 2 DICOM I had trouble with, but the 3rd is giving this error now: 

```
~/Dropbox/AAA_MGB_job/repos/highdicom/src/highdicom/pr/sop.py in __init__(self, referenced_images, series_instance_uid, series_number, sop_instance_uid, instance_number, manufacturer, manufacturer_model_name, software_versions, device_serial_number, content_label, content_description, graphic_annotations, graphic_layers, graphic_groups, concept_name, institution_name, institutional_department_name, content_creator_name, content_creator_identification, modality_lut_transformation, voi_lut_transformations, presentation_lut_transformation, transfer_syntax_uid, **kwargs)
    293             )
    294         else:
--> 295             voi_lut_transformations = _get_softcopy_voi_lut_transformations(
    296                 referenced_images
    297             )

~/Dropbox/AAA_MGB_job/repos/highdicom/src/highdicom/pr/content.py in _get_softcopy_voi_lut_transformations(referenced_images)
   1508 
   1509             transformations.append(
-> 1510                 SoftcopyVOILUTTransformation(
   1511                     window_center=center,
   1512                     window_width=width,

~/Dropbox/AAA_MGB_job/repos/highdicom/src/highdicom/pr/content.py in __init__(self, window_center, window_width, window_explanation, voi_lut_function, voi_luts, referenced_images)
    755 
    756         """  # noqa: E501
--> 757         super().__init__(
    758             window_center=window_center,
    759             window_width=window_width,

~/Dropbox/AAA_MGB_job/repos/highdicom/src/highdicom/content.py in __init__(self, window_center, window_width, window_explanation, voi_lut_function, voi_luts)
   1857                     (len(window_explanation) != len(window_center))
   1858                 ):
-> 1859                     raise ValueError(
   1860                         'Length of "window_explanation" must match length of '
   1861                         '"window_center".'

ValueError: Length of "window_explanation" must match length of "window_center".
```

This DICOM doesn't have the "WindowExplanation" attribute and it looks like HighDicom is trying to create one (?)
